src/components/common/zoom_button/zoom_button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+// This is a function component that will provided 
+// the user with a button to zoom in or out incase 
+// they don't have a mouse or keyboard option.
+
+export const ZoomButton = ({type, handleViewportChange, currentZoom}) => (
+    <span onClick ={() =>handleViewportChange({zoom:type === '+' ? currentZoom+=0.5: currentZoom-=0.5})}
+    >{type}</span>
+
+
+)
```

Comment:
@Sal-Meza another good practice is to ensure you leave the newline character at the end of the file: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

There's actually a great macro most dev's use which essentially inserts the newline (if accidentally removed) in VSCode: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files